Assets/GameEngine.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEngine;
+
+public class GameEngine : MonoBehaviour {
+
+    public GameObject unitPrefab;
+    private Transform source;
+
+	public void RegisterStrongholdClick(Transform selected) {
+        if (source == null) {
+            // If source is not yet selected, selected stronghold is the source
+            source = selected;
+        } else {
+            ObjectOwnerState sourceInfo = source.GetComponent<ObjectOwnerState>();
+
+            // If source is owned by a player, and not the same as target, send unit
+            if (sourceInfo.player != null && source != selected) {
+                SendUnit(sourceInfo, selected);
+            }
+
+            source = null;
+        }
+    }
+
+    private void SendUnit(ObjectOwnerState sourceInfo, Transform selected) {
+        // Create unit at source stronghold
+        GameObject unit = Instantiate(unitPrefab, source.position, Quaternion.identity);
```

Comment:
What is Quaternion.identity?

Reply:
`Quaternion` is used for rotation. [`Quaternion.identity`](https://docs.unity3d.com/ScriptReference/Quaternion-identity.html) means no rotation.